astropy/time/formats.py

```diff
@@ -6,7 +6,7 @@
 import fnmatch
 import time
 import re
-from datetime import datetime
+from datetime import datetime, tzinfo, timedelta
```

Comment:
At this point it might be better to just `import datetime` as a module.  I found myself wondering what `timedelta` was when reading code below, so I think it would be more clear to see `datetime.timedelta`.